<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="client_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="project_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="issued_tree">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>settings</string>
    </property>
   </widget>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
